karmada-CLI-installtion-and-deploy-example/index.json

```diff
@@ -0,0 +1,38 @@
+{
+    "title": "install Karmada cluster through CLI",
```

Comment:
We don't usually say `Karmada cluster`. How about 
```suggestion
    "title": "install Karmada through CLI",
```

Reply:
For the first question，We define the name of each step in the index. json file。like this：
![image](https://user-images.githubusercontent.com/35321989/229688018-2596ea11-4fdb-4bdd-a853-f5b9f30ef6f2.png)